src/features/device-state/index.ts

```diff
@@ -45,6 +46,16 @@ export const setup = (app: Application) => {
 		statePatchV2,
 	);
 	app.patch('/device/v3/state', middleware.authenticatedApiKey, statePatchV3);
+	app.get(
+		'/device/v3/fleet/:fleetUuid/state',
+		middleware.authenticated,
+		fleetStateV3,
+	);
+	app.get(
+		'/device/v3/fleet/:fleetUuid/release/:releaseUuid/state',
+		middleware.authenticated,
+		fleetStateV3,
+	);
```

Comment:
Given that the releaseUuid is used like an optional parameter shouldn't we just make it an optional query parameter rather than having 2 different endpoints?
On principle having `/device/v3/fleet/:fleetUuid/state` & `/device/v3/fleet/:fleetUuid/release/:releaseUuid/state` returning the same result sounds a bit sketchy. I also think that from REST semantics pov `/device/v3/fleet/:fleetUuid/release/:releaseUuid/state` might not be that correct, since it sounds like we are asking for the target state of the release of that fleet, while what we really are asking is the target state of the fleet when configured to run the provided release (imo that points that the release should be a parameter).